### PR TITLE
Update README

### LINKED ISSUE #2: provide detailed readme
since this module is fairly arcane, we should provide a detailed explanation of why it exists and what problems it fixes

I would also appreciate examples or a demonstration with terminal output (text is just fine)

### PR DESCRIPTION
**Pull request by nforro**
Fixes #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+Python RPM shim module for use in virtalenvs.
+
+## Purpose
+
+RPM Python bindings are tied to system RPM installation and are not available as a Python package (on PyPI or elsewhere). This shim module makes it possible to import and use the bindings in a virtualenv.
+
+There is no point installing this shim module on a bare system, outside of a virtualenv. It should still work, but there is no benefit. If you want to do that anyway, pay attention not to overwrite installed RPM Python bindings.
+
+## Example
+
```
Comment (TomasTomecek) [inline README.md:12]:
```suggestion

Here is a scenario of how this module enables usage of RPM Python bindings in a newly created virtualenv. First commands are run on a host system.
```

**Review by TomasTomecek — APPROVED** [review]
nice and clear, thank you!

I just added a bit of context to the example, without the intro, I had to spend some time understand the whole context.

File: README.md (hunk 1 above)
Comment (nforro) [reply README.md:12]: Thanks, updated.

**Comment by nforro** [comment]
Merging manually.

**Pull request merged**